### ISSUE TITLE
Generate low R signatures

### ISSUE THREAD

**Issue by nwasiq** [bitcoinj/bitcoinj#2125]
Is there any plan to support low R signatures like with NBitcoin?

**Comment by schildbach** [comment]
Yes. I looked at this yesterday, and something on a private branch.

**Comment by schildbach** [comment]
At the moment I'm hesitant to go forward on this.

1. Changing how the K values are generated has some risk of it going wrong, and as a consequence stolen Bitcoins.
2. Taproot makes this all pointless, as soon as we support it. So we should perhaps put our time into Taproot.

For the reference, there is a good explanation of the idea: https://bitcoinops.org/en/topics/low-r-grinding/

**Comment by msgilligan** [comment]
This feature is in secp256k1-jdk which we are planning to integrate into bitcoinj for the 0.19 release.
